darts/datasets/__init__.py

```diff
@@ -308,3 +309,130 @@ def __init__(self):
+    .. [2] https://arxiv.org/abs/2012.07436
+    """
+
+    def __init__(self):
+        super().__init__(
+            metadata=DatasetLoaderMetadata(
+                "ETTh1.csv",
+                uri="https://raw.githubusercontent.com/zhouhaoyi/ETDataset/main/ETT-small/ETTh1.csv",
+                hash="8381763947c85f4be6ac456c508460d6",
+                header_time="date",
+                format_time="%Y-%m-%d %H:%M:%S",
+            )
+        )
+
+
+class ETTh2Dataset(DatasetLoaderCSV):
+    """
+    The data of 2 Electricity Transformers at 2 stations, including load, oil temperature.
+    The dataset ranges from 2016/07 to 2018/07 taken hourly.
+    Source: [1][2]_
+
+    References
+    ----------
+    .. [1] https://github.com/zhouhaoyi/ETDataset
+    .. [2] https://arxiv.org/abs/2012.07436
+    """
+
+    def __init__(self):
+        super().__init__(
+            metadata=DatasetLoaderMetadata(
+                "ETTh2.csv",
+                uri="https://raw.githubusercontent.com/zhouhaoyi/ETDataset/main/ETT-small/ETTh2.csv",
+                hash="51a229a3fc13579dd939364fefe9c7ab",
+                header_time="date",
+                format_time="%Y-%m-%d %H:%M:%S",
+            )
+        )
+
+
+class ETTm1Dataset(DatasetLoaderCSV):
+    """
+    The data of 2 Electricity Transformers at 2 stations, including load, oil temperature.
+    The dataset ranges from 2016/07 to 2018/07 recorded every 15 minutes.
+    Source: [1][2]_
+
+    References
+    ----------
+    .. [1] https://github.com/zhouhaoyi/ETDataset
+    .. [2] https://arxiv.org/abs/2012.07436
+    """
+
+    def __init__(self):
+        super().__init__(
+            metadata=DatasetLoaderMetadata(
+                "ETTm1.csv",
+                uri="https://raw.githubusercontent.com/zhouhaoyi/ETDataset/main/ETT-small/ETTm1.csv",
+                hash="82d6bd89109c63d075d99c1077b33f38",
+                header_time="date",
+                format_time="%Y-%m-%d %H:%M:%S",
+            )
+        )
+
+
+class ETTm2Dataset(DatasetLoaderCSV):
+    """
+    The data of 2 Electricity Transformers at 2 stations, including load, oil temperature.
+    The dataset ranges from 2016/07 to 2018/07 recorded every 15 minutes.
+    Source: [1][2]_
+
+    References
+    ----------
+    .. [1] https://github.com/zhouhaoyi/ETDataset
+    .. [2] https://arxiv.org/abs/2012.07436
+    """
+
+    def __init__(self):
+        super().__init__(
+            metadata=DatasetLoaderMetadata(
+                "ETTm2.csv",
+                uri="https://raw.githubusercontent.com/zhouhaoyi/ETDataset/main/ETT-small/ETTm2.csv",
+                hash="7687e47825335860bf58bccb31be0c56",
+                header_time="date",
+                format_time="%Y-%m-%d %H:%M:%S",
+            )
+        )
+
+
+class ElectricityDataset(DatasetLoaderCSV):
+    """
+    Measurements of electric power consumption in one household with a one-minute sampling rate over a period of almost
```

Comment:
A few things seem off here. The time granularity is 15-mins and not minute, and also, looking at the dataset description on UCI ML repository, there is "one column per client". I understand this as meaning there are 370 distinct housholds, and this is their electricity consumption (vs 370 different electrical quantities of one household). Let me know if I misunderstood this? Also, I think that in this case we should probably return a list of 370 `TimeSeries` rather than a single `TimeSeries` - I somehow expect this to be a more natural representation (each household being one "data point" -vs- one data point of 370 dimensions). Let me know what you think.

Reply:
Im not sure where I found that description, but it is definitely wrong. I have updated it with the correct information. 

I think the list vs. single `TimeSeries` is related to the multi-series vs. multivariate discussion. As a small anecdote, I get better results with the ice cream & heater dataset if I `fit` using the dataset as a multivariate, even if Im making univariate predictions.

In general, this feels like a decision that should be left to the user. The tutorials show how to turn quickly turn a multivariate dataset into a list of `TimeSeries`. I could add some example code in the docstring.

Reply:
OK fair enough we can leave it like that. It'd be awesome if you add an example code in the docstring - making it clear what is returned is a single multivariate `TimeSeries` containing 370 components, which can be transformed to 370 univariate `TimeSeries` (from experience it's a constant source of confusion among users).

Reply:
I added some sample code. let me know what you think